docs/internals/contributing/triaging-tickets.txt

```diff
@@ -2,16 +2,16 @@
 Triaging tickets
 ================
 
-Django uses Trac_ for managing the work on the code base. Trac is a
+Django uses Trac_ for managing the work on the codebase. Trac is a
```

Comment:
I would revert; not sure that 'codebase' is a word.

Reply:
Depends on the source ;) 

Here's a mainstream English dictionary:
<img width="432" alt="Screen Shot 2021-12-14 at 4 09 52 PM" src="https://user-images.githubusercontent.com/38668450/146079814-c4f9b3b4-30ac-4e7e-ba5e-eb61e0068641.png">

Reply:
Code base / codebase is not in the style guide. Source code is though!

Reply:
changing it to source code then :)